documentation/_data/sidebars/home_sidebar.yml

```diff
@@ -55,6 +55,9 @@ entries:
       - title: Best Practices
         url: /bestpractices.html
         output: web
+      - title: Maximum supported scale limit

```

Comment:
title is wrong. looks like a copy paste.

Reply:
I have modified it.